### PR TITLE
MM-40845 Fix the menu button opening too far right.

### PR DESCRIPTION
**Pull request by crspeller**
#### Summary
Fix the dot menus floating too far to the right.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-40845

### CODE REVIEW

File: webapp/src/components/dot_menu.tsx [hunk 1]
```diff
@@ -57,7 +57,6 @@ export const DropdownMenu = styled.div<DropdownMenuProps>`
     list-style: none;
 
     padding: 10px 0;
-	margin-left: 50px;
```
Comment (lieut-data) [inline webapp/src/components/dot_menu.tsx:60]: That happened to actually be my first fix too... but I was worried it was added to fix something else?

**Review by lieut-data — APPROVED** [review]

File: webapp/src/components/dot_menu.tsx (hunk 1 above)
Comment (crspeller) [reply webapp/src/components/dot_menu.tsx:60]: I thought so too, but I checked the other cases and it seemed fine without it.

Comment (cpoile) [reply webapp/src/components/dot_menu.tsx:60]: I looked at the blame, and it says you added it yesterday? Do you remember why?

**Review by cpoile — APPROVED** [review]

File: webapp/src/components/dot_menu.tsx (hunk 1 above)
Comment (crspeller) [reply webapp/src/components/dot_menu.tsx:60]: It came in with the advanced permissions stuff. I don't remember why.

**Review by calebroseland — APPROVED** [review]

**Pull request merged**